### ISSUE TITLE
debuginfo package does not contain debug library

### ISSUE THREAD

**Issue by kmaehashi** [jubatus/jubatus#970]
`jubatus-core-debuginfo-0.1.1-1.el6.x86_64.rpm` only has 1,756 bytes, which is almost empty.
I'm still investigating on this.

http://download.jubat.us/yum/rhel/6/stable/x86_64/

**Comment by kmaehashi** [comment]
This was a waf bug, which got fixed in waf-1.7.0.
https://github.com/waf-project/waf/commit/284c3a0810412fe5abd62fec51df7c8c3fe0724f

* We introduced vnum (SONAME) in Jubatus 0.6.1. (#812)
* waf installs shared libraries with SONAME as permission 0644. (c.f., shared libraries without SONAME are installed as permission 0755)
* rpmbuild does not generate debuinfo for shared libraries without execution permission.

So, debuginfo package was empty since Jubatus 0.6.1.

Note: this does not affect the functionality of Jubatus.
